zipline/testing/core.py

```diff
@@ -761,7 +763,8 @@ class tmp_assets_db(object):
     """
     _default_equities = sentinel('_default_equities')
 
-    def __init__(self, equities=_default_equities, **frames):
+    def __init__(self, url='sqlite://', equities=_default_equities, **frames):
```

Comment:
Was going off of what we had before, as well the SQLAlchemy docs, http://docs.sqlalchemy.org/en/latest/core/engines.html#sqlite

I'm fine with changing to make the use of `:memory:` more explicit, though, will change here and in `tmp_asset_finder` and `WithAssetFinder`.